### PR TITLE
examples: Update resource quota README

### PR DESCRIPTION
**Pull request by kargakis**
@derekwaynecarr @deads2k  PTAL

### CODE REVIEW

File: examples/project-quota/README.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ Resource limits:
 
 You can now see that the project has set min/max limits at the pod and container scopes.
 
-If a pod is created that has no cpu resource limit set, the default (100m) will be set as an explicit limit.  Similarly, if a pod is created that has no memory resource limit set, the default (256Mi) will be set as an explicit limit.
+If a pod is created that has no cpu resource limit set, the default (100m) will be set as an explicit limit.  Similarly, if a pod is created that has no memory resource limit set, the default (100Mi) will be set as an explicit limit.
```
Comment (deads2k) [inline examples/project-quota/README.md]: feeling lazy.  Can you point me to this default in the code?

Comment (kargakis) [inline examples/project-quota/README.md]: L199 in this doc

Comment (kargakis) [inline examples/project-quota/README.md]: or L162

**Comment by deads2k** [comment]
[merge]

**Pull request merged**